curriculum/challenges/english/14-responsive-web-design-22/learn-css-flexbox-by-building-a-photo-gallery/6493bc0d99879635209565aa.md

```diff
@@ -0,0 +1,85 @@
+---
+id: 6493bc0d99879635209565aa
+title: Step 8
+challengeType: 0
+dashedName: step-8
+---
+
+# --description--
+
+Ooh you have a problem. See how the images overflow the container? That's because the `box-sizing property` is at the `content-box` value, aka the default. This doesn't include the padding and border when calculating an element's size. Now, to solve this issue, use `border-box`, the box-sizing property's second value. Unlike `content-box`, `border-box` excludes the margins and only adds the padding and border in calculating the size of the element. Try changing the box-sizing value from one to the other, and see the difference.
+Lastly, give the img elements a uniform height of 300px.
+
+
+# --hints--
+
+
+Your * selector should have a `box-sizing` property set to `border-box` as the value.
+
+```js
+assert( new __helpers.CSSHelp(document).getStyle('*')?.boxSizing === 'border-box');
+```
+
+# --seed--
+
+## --seed-contents--
+
+```html
+<!DOCTYPE html>
+# --seed--
```

Comment:
i think here is the problem that makes tests to fail -
each challenge file must have only one `# --seed--` region
and also only one `## --seed-contents`...
here you have it twice...
you must remove the duplicate ones

Reply:
Done @shootermv